pt/lessons/specifics/plug.md

```diff
@@ -271,9 +271,9 @@ plug(
 Nós automaticamente invocamos `VerifyRequest.init(fields: ["content", "mimetype"],
 paths: ["/upload"])`. Isso por sua vez passa as opções recebidas para a função `VerifyRequest.call(conn, opts)`.
 
-Let's take a look at this plug in action! Go ahead and crash your local server (rember, that's done by pressing `ctrl + c` twice). Then restart the server (`mix run --no-halt`).
-Now go to <http://127.0.0.1:8080/upload> in your browser and you'll see that the page simply isn't working. We're not even getting our 'Oops!' message. Now let's add our required params by going to <http://127.0.0.1:8080/upload?content=thing1&mimetype=thing2>. Now we should see our 'Uploaded' message.
-It's not great that when we throw an error we don't get _any_ page, but we'll deal with how to handle errors with plugs later.
+Vamos ver como esse plug funciona em ação! Vá em frente e quebre seu servidor local (lembre-se, isso pode ser feito pressionando `ctrl + c` duas vezes). Então reinicie o servidor com (`mix run --no-halt`).
+Agora acesse <http://127.0.0.1:8080/upload> no seu navegador e você vai ver como a página simplesmente não está funcionando. Nós não estamos recebendo nossa mensagem 'Oops!'. Agora vamos adicionar os parâmetros obrigatórios por acessar <http://127.0.0.1:8080/upload?content=thing1&mimetype=thing2>. Agora nós devemos ver nossa mensagem 'Uploaded'. 
+Não é legal não receber _nenhuma_ página caso um erro occura, mas nós vamos como lidar com erros em plugs depois.
```

Comment:
There is a typo here: "... caso um erro **occura**"=> "ocorra".

Also, I think the end of the phrase could be translated to: "mas nós vamos lidar com como tratar erros com plug depois.".

Reply:
@anderkonzen thanks for the review! Sorry for the delay, I just pushed the suggested changes. Could you please take another look?

Reply:
There is still one tiny typo: `ocurra` => `ocorra`

The rest is OK!

Reply:
@anderkonzen :man_facepalming: sorry :joy: :joy:. I will fix it right now! Thanks!

Reply:
@anderkonzen done.